controllers/nginx/pkg/template/template.go

```diff
@@ -658,3 +661,24 @@ func buildForwardedFor(input interface{}) string {
 	ffh = strings.ToLower(ffh)
 	return fmt.Sprintf("$http_%v", ffh)
 }
+
+func trustHTTPHeaders(input interface{}) bool {
+	conf, ok := input.(config.TemplateConfig)
+	if !ok {
+		return true
```

Comment:
(Continuing from my comment below)
Maybe ok is always a non-zero value here.
Could you check if input.(config.TemplateConfig) is returning conf correctly? Or maybe I'm missing something..

Reply:
fixed